config/hardware/extruder/lgx_lite.cfg

```diff
@@ -9,6 +9,7 @@ full_steps_per_rotation: 200
 nozzle_diameter: 0.400
 filament_diameter: 1.75
 max_extrude_only_distance: 110
+sensor_type: ATC Semitec 104GT-2
```

Comment:
Just for the note, temperature sensor is not something related to the lgx lite. But for sure klipper mixes that into the same extruder section. I would prefer to move that elsewhere (it's a pure user definition, as none has the same one). But as we need to do that everywhere, I'm fine with that for now

Reply:
Yes, maybe gather all the sensor type in the same file maybe a solution, but given the great variability of the configs for this type of component it will most certainly end up in the overrides...